101-VM-Virus-Attack-Prevention/nested/microsoft.loganalytics/workspaces.json

```diff
@@ -0,0 +1,113 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "omsWorkspaceName":{
+            "type": "string",
+            "defaultValue": "qwerty-workspace"
+        },
+        "omsSolutionsName":{
+            "type": "array"
+        },
+        "tags": {
+            "type": "object"
+        },
+        "sku": {
+            "type": "string",
+            "allowedValues": [
+              "Free",
+              "Standalone",
+              "PerNode"
+            ],
+            "defaultValue": "Free",
+            "metadata": {
+              "description": "Service Tier: Free, Standalone, or PerNode"
+            }
+        },
+        "dataRetention": {
+            "type": "int",
+            "defaultValue": 90,
+            "minValue": 7,
+            "maxValue": 730,
+            "metadata": {
+                "description": "Number of days of retention. Free plans can only have 7 days, Standalone and OMS plans include 30 days for free"
+            }
+        },
+        "location": {
+            "type": "string",
+            "allowedValues": [
+                "East US",
+                "West Europe",
+                "Southeast Asia",
+                "Australia Southeast"
+            ],
+            "defaultValue": "East US"
+        }
+    },
+    "resources": [
+        {
+            "name": "[parameters('omsWorkspaceName')]",
+            "type": "Microsoft.OperationalInsights/workspaces",
+            "apiVersion": "2017-03-15-preview",
+            "location": "[parameters('location')]",
+            "tags": "[parameters('tags')]",
+            "properties": {
+              "sku": {
+                "name": "[parameters('sku')]"
+              },
+              "retention": "[parameters('dataRetention')]"
+            }
+        },
+        {
+            "id": "[concat(resourceGroup().id, '/providers/Microsoft.OperationsManagement/solutions/', parameters('omsSolutionsName')[copyIndex()], '(', parameters('omsWorkspaceName'), ')')]",
+            "type": "Microsoft.OperationsManagement/solutions",
+            "name": "[concat(parameters('omsSolutionsName')[copyIndex()], '(', parameters('omsWorkspaceName'), ')')]",
+            "apiVersion": "2015-11-01-preview",
```

Comment:
update apiVersion

Reply:
updated same as Microsoft.OperationalInsights/workspaces

Reply:
I don't see the change here...

Reply:
or here...